tasks/main.yml

```diff
@@ -113,8 +113,36 @@
     name: "{{ camac_releasedir }}/camac/django/"
     extra_args: "--upgrade"
 
+- name: reset django_migrations table
+  shell: |
+    psql \
+        -h {{ camac_db_host }} \
+        -p {{ camac_db_port }} \
+        -U {{ camac_db_username }} \
+        -d {{ camac_db_dbname }} \
+        -c "TRUNCATE django_migrations"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
```

Comment:
@czosel @lucasb Are the precautions of having a boolean flag as well as defined dictionary enough?

Reply:
The only possible alternative i could think of is to implement a check for the existence of a given migration - then leaving the setting in too long shouldn't cause any issues. Not sure how much effort that would be though?
/cc @sliverc

Reply:
it would properly be better to check for a given migration in the table (one which is only available in the camac-be project) and only then truncate.

Reply:
The when check is enough like this if it here so the user can activate the feature for a ansible run as needed. If we want to automate setting it we would need to consult what is already installed and possibly also the state in the database.

I expect this to be run in an ad-hoc fashion by ops so I don't think it's really worth automating it too much, whomever uses it will probably set in on the command line in an ansible run.

Reply:
I've planned to remove the fake migration logic after the next deployment. It's a one off.